soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -326,7 +326,7 @@ void Randomizer::LoadMerchantMessages() {
     CustomMessageManager::Instance->CreateMessage(
         Randomizer::merchantMessageTableID, TEXT_BEAN_SALESMAN_BUY_FOR_100,
         CustomMessage("I never thought I'd say this, but I'm &selling the last %rMagic Bean%w. %r99%w Rupees...\x1B&%gYes&No%w",
-                      "Ich hätte nie gedacht, daß ich das sage, aber ich &verkaufe die letzte %rWundererbse%w für %r99%w Rubine.\x1B&%gJa&Nein%w",
+                      "Ich hätte nie gedacht, daß ich das&sage, aber ich verkaufe die&letzte %rWundererbse%w für %r99%w Rubine.\x1B&%gJa&Nein%w",
```

Comment:
OK this is my fault for bad advice, but thanks to this I just learned that OoT does not like 3 line + yes-no selection textboxes. it might just have to be the second text box.
![image](https://github.com/user-attachments/assets/b034d292-ec65-4226-9350-aeb2aa12818f)

Reply:
We don't make mistakes, just happy little accidents.